.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+//SET ASID TO ONE YOU WERE GIVEN
```

Comment:
I understand this is not in `.gitignore` to act as a sample environment?
How this will work if people will add their own environments? I think there should be a README.MD describing how to use this VSCode HTTP Client and environments with this as sample content